### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by muhammedb8826**
I have added three files index.html, style.css, and test.dm
I have installed lineters and stylelint packages for the project

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @muhammedb8826   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [x] Kindly make the `test.md` file ignored by git so it must not be represented in the repo and it must be presented in `gitignore` file. :wink:  

- [x] Kindly make sure to update a README file by copying the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and make sure to customize it according to your project. :wink: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure only next time the changes you made must be in the `feature` branch and must be not represented In the `main` branch until it's merged.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muhammedb8826** [comment]
Thank you so much

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @muhammedb8826, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Linters passed ✔️
- GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [x] I would advise you to use a well-detailed readme file template provided by Microverse, it would help you and the visitors to understand what the repo entails at a glance. You can copy the file [here](https://github.com/microverseinc/readme-template) and adapt it to your own. Emphasis on adapt it to your own, dont remove any section, just replace them all to fit your project.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+- lighthouse (<code>.github/workflows/linters.yml</code>)
+- webhint (<code>.github/workflows/linters.yml</code>)
+- stylelint (<code>.github/workflows/linters.yml</code>)
+
+### Run tests
+After pushing the changes on github, merge your pull request and let github action process changes.
+
+### Deployment
+Check for the tests when you generate a pull request and fix errors, if any.
+For stylelint errors run:<code>sudo npx stylelint "**/*.{css,scss}" --fix</code> for automatic fix.
```
Comment (Felix45) [inline README.md:57]: - [ Optional ] The `Getting Started` section of your readme file is very important, this means you always have to fill it correctly. Please see an example of how I would fill in this section in my `readme` file, the instructions you give in the `readme` file should always be clear and easy for someone to follow.



## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

```sh
   git clone git@github.com:muhammedb8826/Hello-world.git
```

### Prerequisites

In order to run this project you need:
 - NodeJS
 - Browser

### Setup

Clone this repository to your desired folder:



```sh
  git clone git@github.com:muhammedb8826/Hello-world.git
  cd Hello-World
```

### Install

Install project dependencies in this project with:

```sh
  npm install
```

### Usage

To run the project, open the project with live server and visit [http://localhost:5500](http://localhost:5500)

### Open in browser

http://localhost:5500

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @muhammedb8826  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] Implemented all the changes from the previous reviewer :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by muhammedb8826** [comment]
Awesome, thank you so much!

**Pull request merged**